### PR TITLE
events: add Art of Scala 2023

### PR DESCRIPTION
**Pull request by SethTisue**

### CODE REVIEW

**Comment by SethTisue** [comment]
rendered locally

Screenshot 2023-10-25 at 9 45 53 AM

File: _events/2023-11-17-art-of-scala.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+category: event
+title: The Art of Scala
```
Comment (Philippus) [inline _events/2023-11-17-art-of-scala.md:3]: The site doesn't say "The"
```suggestion
title: Art of Scala
```

File: _events/2023-11-17-art-of-scala.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+logo: /resources/img/artofscala.png
+location: Warsaw, Poland
+description: Unique non-commercial conference for Scala engineers and enthusiasts.
+start: 13 November 2023
+end: 13 November 2023
```
Comment (Philippus) [inline _events/2023-11-17-art-of-scala.md:8]:
```suggestion
start: 17 November 2023
end: 17 November 2023
```

**Review by Philippus — CHANGES_REQUESTED** [review]

**Comment by SethTisue** [comment]
gah, thanks @Philippus

**Comment by SethTisue** [comment]
https://www.scala-lang.org/events/

**Pull request merged**